web-common/src/features/dashboards/leaderboard/LeaderboardContextColumnMenu.svelte

```diff
@@ -26,6 +26,8 @@
       metricsExplorerStore.displayDeltaChange(metricViewName);
     } else if (key === LeaderboardContextColumn.PERCENT) {
       metricsExplorerStore.displayPercentOfTotal(metricViewName);
+    } else if (key === LeaderboardContextColumn.DELTA_ABSOLUTE) {
```

Comment:
This if-else chain can be moved over to switch-case. Easy to add more options. Applies to code in LeaderboardHeader and leaderboard-utils as well.